src/muya/lib/contentState/imageCtrl.js

```diff
@@ -0,0 +1,135 @@
+import { URL_REG } from '../config'
+
+const imageCtrl = ContentState => {
+  /**
+   * insert inline image at the cursor position.
+   */
+  ContentState.prototype.insertImage = function ({ alt = '', src = '', title = '' }) {
+    const match = /\/?([^./]+)\.[a-z]+$/.exec(src)
```

Comment:
Do you mean filename in Windows is not shown in pathname?

Reply:
I don't event tested the PR on Windows but what does the regex match? If you try to match `/a/b/<match[1]>.ext` it will not work on Windows because Windows using `\` instead of `/`.